packages/webcert/src/store/certificate/certificateMiddleware.test.ts

```diff
@@ -590,6 +593,33 @@ describe('Test certificate middleware', () => {
       expect(createdCertificateId).toEqual(response.certificateId)
     })
   })
+
+  describe('Handle clear certificate data element value', () => {
+    it('should clear boolean value', async () => {
+      const certificate = getCertificateWithShowValidationAndBooleanValue(false)
+      testStore.dispatch(updateCertificate(certificate))
+      testStore.dispatch(clearCertificateDataElementValue(certificate.data[0].id))
+
+      await flushPromises()
+
+      expect((testStore.getState().ui.uiCertificate.certificate.data[0].value as ValueBoolean).selected).toBe(null)
+    })
+    it('should clear date value', async () => {
+      const certificate = getCertificateWithShowValidationAndDateValue('2022-01-01')
+      testStore.dispatch(updateCertificate(certificate))
+      testStore.dispatch(clearCertificateDataElementValue(certificate.data[0].id))
+
+      await flushPromises()
+
+      expect((testStore.getState().ui.uiCertificate.certificate.data[0].value as ValueDate).date).toBeFalsy()
+    })
+  })
+
+  const setDefaultUser = () => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    testStore.dispatch(updateUser({ signingMethod: SigningMethod.DSS }))
```

Comment:
Looks like this is not used so should probably be removed, but if it was used it could be solved with `testStore.dispatch(updateUser({ ...getUser(), signingMethod: SigningMethod.DSS }))` without having to disable type checking.